src/main/java/org/sonarlint/intellij/ui/nodes/IssueNode.java

```diff
@@ -25,15 +25,19 @@
 import com.intellij.openapi.fileEditor.FileDocumentManager;
 import com.intellij.ui.ColoredTreeCellRenderer;
 import com.intellij.ui.SimpleTextAttributes;
-import org.sonarlint.intellij.issue.IssuePointer;
-import org.sonarlint.intellij.util.ResourceLoader;
 
-import javax.annotation.Nonnull;
 import java.io.IOException;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.Nonnull;
+
+import org.sonarlint.intellij.issue.IssuePointer;
+import org.sonarlint.intellij.util.ResourceLoader;
 
 public class IssueNode extends AbstractNode {
   private static final Logger LOGGER = Logger.getInstance(IssueNode.class);
   private final IssuePointer issue;
+  private String timeLabel;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "timeLabel" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1068)